pyintegration/deephaven2/table.py

```diff
@@ -89,6 +89,12 @@ def __repr__(self):
     def get_dh_table(self):
         return self._j_table
 
+    def write_csv(self: Table, path: str, cols: List[str] = []) -> None:
+        try:
+            _JTableTools.writeCsv(self._j_table, path, *cols)
+        except Exception as e:
+            raise DHError("write csv failed.") from e
+
```

Comment:
I don't think we want this method on the table.  We support many data formats and have finally started packaging each format into its own module.  By having this method here, we have decided that CSV is somehow more special than the other formats, and we are willing to add it to the table API.  I think the method in the CSV module is sufficient.  If it isn't, we can easily add a method here in the future.  It is harder to take a method away from here.